tmux-modal.tmux

```diff
@@ -152,6 +152,13 @@ elif ! parse_yn_opt "$SHOW_CMD_KEYS_VAL" $SHOW_CMD_KEYS_OPT; then
     exit 22
 fi
 
+# Modal mode status bar icon when key tables are in use
```

Comment:
I think "... when key tables are in use" is a bit redundant. Can remove this (and end sentence with period as everywhere else).